sentry/src/test/java/io/sentry/GsonSerializerTest.kt

```diff
@@ -472,4 +499,17 @@ class GsonSerializerTest {
             "debug",
             "io.sentry@1.0+123"
         )
+
+    private val userFeedback: UserFeedback get()  {
+        val eventId = SentryId("c2fb8fee2e2b49758bcb67cda0f713c7")
+        val userFeedback = UserFeedback(eventId)
```

Comment:
```suggestion
        val userFeedback = UserFeedback(eventId).apply {
```

Reply:
Ah yes, nice. It's been a while since I coded in Kotlin. I miss it a lot. 🥇